modules/custom/entity_access_by_field/src/EntityAccessHelper.php

```diff
@@ -21,19 +21,30 @@ class EntityAccessHelper {
    * @return array
    *   An array containing a list of values to ignore.
    */
-  public static function getIgnoredValues() {
+  public static function getIgnoredValues(): array {
     return [];
```

Comment:
Hi @navneet0693 ,

Already discussed this a bit with @ronaldtebrake , can you think of any reason why we need this helper function as it merely returns an empty array?

Reply:
@sjoerdvandervis I am not very sure on this and haven't touched the internal code. The changes were introduced in https://github.com/goalgorilla/open_social/pull/422 which was specifically changed in this commit: https://github.com/goalgorilla/open_social/pull/422/commits/09e071c7d6b939471a686272a89134ed8c53a983. Also I see a TODO added https://github.com/goalgorilla/open_social/blob/8.x-8.x/modules/custom/entity_access_by_field/src/EntityAccessHelper.php#L19